docs/docs/workflows/create.md

```diff
@@ -45,7 +45,7 @@ It's recommended to use Terraform for infrastructure management, but you can use
 
 :::info
 
-  When using Terraform, you can use the [Terraform module](./terraform-module.md) for ease of use to manage the entire Constellation cluster.
+  When using Terraform, you can use the [Terraform provider](./terraform-provider.md) for ease of use to manage the entire Constellation cluster.
```

Comment:
I have no opinion on what advice we should give here. I think most of the users will know what they want to do either way. Constellation is not a product that you just try out in your Homelab because you feel like it. Therefore we should keep it somewhat technical here I think. I also don't understand why having this hint in the self-managed infrastructure tab changes anything on the suggestion. But I'm not informed enough on what we want to convey in our docs here. Maybe @thomasten can take a look at this and share his thoughts.

Reply:
I agree with Adrian that users looking for self-managed (or Terraform-managed to be precise) probably also prefer to use the provider instead of the CLI, so I don't consider "easier to use" that bad here. However, it doesn't have much informational value to me, so I'd prefer Moritz's suggested change.